client-src/src/pages/home/home.ts

```diff
@@ -29,11 +29,11 @@ export class HomePage {
   }
 
   goToCounselorDash(){
```

Comment:
Instead of multiple goto methods, create a generic switchService method and can pass service name as param. Now this may have switchCase. This will be better maintainable and would avoid duplicating the code. Method may have params as serviceName, isRoot.

Reply:
Just for the welcome screen or the whole app?

Reply:
This should be for the whole app. Can be lower priority for now. Add task. Approving this for now.

Reply:
This is this the ionic way to navigate, moving it to a separate service will create unnecessary abstraction. We are using the navigation service provided by ionic to navigate. If we were to create a separate service it would just be a service providing another service.

Reply:
ok